### PR TITLE
feature/us1900_Changing_General-Giving_to_Ministry_Fund-td reload scripts

### PR DESCRIPTION
**Pull request by lynchl607**
Changing General Giving to Ministry Fund

### CODE REVIEW

File: CI/TestSQL/StandardScripts/02.TestData/07.Donation+Distribution/CreateDonations_DonorsHaveUserAccounts.csv [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
-mpcrds+auto+2@gmail.com,50,8/30/2017,4,2,0,General Giving,1,,,,,,,,,,9/1/2017,,,,(auto) General Giving Two Batches,,
-mpcrds+12@gmail.com,50,8/30/2017,4,2,0,General Giving,1,,,,,,,,,,9/1/2017,,,,(auto) General Giving Two Batches,,
-mpcrds+tremplay.richard@gmail.com,50,8/30/2017,4,2,0,General Giving,1,,,,,,,,,,9/1/2017,,,,(auto) General Giving Two Batches,,
-mpcrds+auto+2@gmail.com,50,8/30/2017,4,2,0,General Giving,1,,,,,,,,,,9/1/2017,,,,(auto) General Giving Two Batches,,
-mpcrds+12@gmail.com,50,8/30/2017,4,2,0,General Giving,1,,,,,,,,,,9/1/2017,,,,(auto) General Giving Two Batches,,
-mpcrds+tremplay.richard@gmail.com,50,8/30/2017,4,2,0,General Giving,1,,,,,,,,,,9/1/2017,,,,(auto) General Giving Two Batches,,
-mpcrds+auto+wilmaflintstone@gmail.com,150,1/3/2018,4,2,1,General Giving,5,,,,,,,0,1,,1/3/2018,,,ch_16odS5Eldv5NE53si5NDZzoN,,,Credit Card donation
+mpcrds+auto+wilmaflintstone@gmail.com,250,9/3/2017,2,2,1,Ministry Fund,5,,,,,,,0,1,,9/3/2017,,,,,,Donation made last year
+mpcrds+auto+2@gmail.com,50,8/30/2017,4,2,0,Ministry Fund,1,,,,,,,,,,9/1/2017,,,,(auto) Ministry Fund Two Batches,,
```
Comment (s-henney) [inline CI/TestSQL/StandardScripts/02.TestData/07.Donation+Distribution/CreateDonations_DonorsHaveUserAccounts.csv]: Could you change "(auto) Ministry Fund Two Batches" back to "(auto) General Giving Two Batches"? This is a batch we create in an earlier script and renaming it here will break the TD/RL script.
Eventually we should update the batche's name, but it requires changing a couple more csv files and our automated tests, so let's leave that for a later PR.

Comment (lynchl607) [reply CI/TestSQL/StandardScripts/02.TestData/07.Donation+Distribution/CreateDonations_DonorsHaveUserAccounts.csv]: Sure I will start on them now.

File: CI/TestSQL/StandardScripts/02.TestData/07.Donation+Distribution/CreateDonations_DonorsHaveUserAccounts.csv [hunk 2]
```diff
@@ -1,43 +1,43 @@
...
 mpcrds+auto+wilmaflintstone@gmail.com,150,1/1/2018,2,2,1,(t+auto) GO Bedrock 2018,5,,,,mpcrds+auto+fredflintstone@gmail.com,,,0,,,,,,,,,Donation to Fred's trip pledge by Wilma - not anonymous.
 mpcrds+auto+wilmaflintstone@gmail.com,150,1/2/2018,2,2,1,(t+auto) GO Bedrock 2018,5,,,,mpcrds+auto+fredflintstone@gmail.com,,,1,,,,,,,,,Donation to Fred's trip pledge by Wilma - anonymous.
-mpcrds+auto+2@gmail.com,50,8/30/2017,4,2,0,General Giving,1,,,,,,,,,,9/1/2017,,,,(auto) General Giving,,
-mpcrds+12@gmail.com,50,8/30/2017,4,2,0,General Giving,1,,,,,,,,,,9/1/2017,,,,(auto) General Giving,,
-mpcrds+tremplay.richard@gmail.com,50,8/30/2017,4,2,0,General Giving,1,,,,,,,,,,9/1/2017,,,,(auto) General Giving,,
-mpcrds+auto+2@gmail.com,50,8/30/2017,4,2,0,General Giving,1,,,,,,,,,,9/1/2017,,,,(auto) General Giving,,
-mpcrds+12@gmail.com,50,8/30/2017,4,2,0,General Giving,1,,,,,,,,,,9/1/2017,,,,(auto) General Giving Two Batches,,
-mpcrds+tremplay.richard@gmail.com,50,8/30/2017,4,2,0,General Giving,1,,,,,,,,,,9/1/2017,,,,(auto) General Giving Two Batches,,
+mpcrds+auto+2@gmail.com,50,8/30/2017,4,2,0,Ministry Fund,1,,,,,,,,,,9/1/2017,,,,(auto) Ministry Fund,,
```
Comment (s-henney) [inline CI/TestSQL/StandardScripts/02.TestData/07.Donation+Distribution/CreateDonations_DonorsHaveUserAccounts.csv]: These too: "(auto) Ministry Fund" should be "(auto) General Giving"

Comment (lynchl607) [reply CI/TestSQL/StandardScripts/02.TestData/07.Donation+Distribution/CreateDonations_DonorsHaveUserAccounts.csv]: Okay I was not sure about them.

**Pull request merged**